src/Badges.t.sol

```diff
@@ -17,14 +17,6 @@ contract ERC1271Mock {
   constructor(bool pass_) {
     pass = pass_;
   }
-
-  function isValidSignature(bytes32 hash, bytes memory signature) public view returns (bytes4) {
```

Comment:
why are we removing this? I think this is needed to check for signatures originating from contracts (liek multisig).. did we remove this test coverage?

Reply:
It was an unused function. It's also unused in the ERC4973 repo